app/main/views/users.py

```diff
@@ -277,6 +278,7 @@ def export_users_for_framework(framework_slug):
             else:
                 application_result = 'pass' if sf.on_framework else 'fail'
             framework_agreement = bool(sf.agreement_returned_at)
+            variations_agreed = ', '.join(sf.agreed_variations.keys()) if sf.agreed_variations else ''
```

Comment:
Cath would be the person to ask, but yeah that seems reasonable, will change now.